auto_nag/topcrash.py

```diff
@@ -206,7 +206,7 @@ def __init__(
         self,
         date: Optional[Union[str, datetime]] = "today",
         duration: Optional[int] = 7,
-        minimum_crashes: Optional[int] = 5,
+        minimum_crashes: Optional[int] = 15,
```

Comment:
Was this change also reflected in the wiki?

Reply:
I just updated the wiki to reflect that:  https://wiki.mozilla.org/index.php?title=CrashKill%2FTopcrash&type=revision&diff=1244767&oldid=1244676